pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "bleak-proglove"
-version = "0.19.0+profilter"
+version = "0.19.4+profilter"
```

Comment:
I guess we can now have a `0.19.4.post1`

Reply:
Yes, yes we can :)